rust/src/pgsql/parser.rs

```diff
@@ -915,9 +967,10 @@ fn parse_row_field(i: &[u8]) -> IResult<&[u8], RowField> {
     ))
 }
 
-pub fn parse_row_description(i: &[u8]) -> IResult<&[u8], PgsqlBEMessage> {
+pub fn parse_row_description(i: &[u8]) -> IResult<&[u8], PgsqlBEMessage, PgsqlParseError<&[u8]>> {
     let (i, identifier) = verify(be_u8, |&x| x == b'T')(i)?;
-    let (i, length) = verify(be_u32, |&x| x > 6)(i)?;
+    //let (i, length) = parse_gt_length(i, 6)?;
```

Comment:
nit: remove commented out code

Reply:
Ugh, that's what I get when having so much back and forth fixing commit history x___x sorry about that and the rust issue x___x